class/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
```

Comment:
- [ ] Good work implementing the Book class, kindly add the method to `add_rental` as it's requested in the requirements 💯